community-groups/sig-governance.md

```diff
@@ -0,0 +1,123 @@
+# SIG Roles and Organizational Governance
+
+This charter adheres to the conventions described in the [Konveyor governance](https://github.com/konveyor/community/blob/main/GOVERNANCE.md).
+It will be updated as needed to meet the current needs of the Konveyor project.
+
+In order to standardize Special Interest Group efforts, create maximum
+transparency, and route contributors to the appropriate SIG, SIGs should follow
+these guidelines:
+
+- Have an approved Charter [SIG charter process]
+- Meet regularly, at least for 30 minutes every 3 weeks, except during holidays
+- Keep up-to-date meeting notes, linked from the SIG's page in the community
+repo
+- Record meetings and make them publicly available on the
+[Konveyor Community YouTube playlist](https://www.youtube.com/channel/UCQ3pW3gSBeCy0tj1J0ub2bw/playlists)  
```

Comment:
What if we started light on recordings and focused recordings on Konveyor Community meetings, but allowed SIGs to meet unrecorded for now?

Reply:
We can certainly do that. But, I am afraid that it might be not transparent. If we go slow on recording, then we should atleast have detailed notes available. wdyt?

Reply:
I was imaging each SIG call will have a running agenda doc open for all, we would track topics there and can update that as needed for notes, but I don't think we need/want to go too far of transcribing/deep notes....would think these are pretty light and most info is captured in GitHub Issues related.

Reply:
That makes sense. So, if someone misses a meeting the only way to catch up would be notes. As long as we communicate that, I think we can slow down on the recorded meetings.

Reply:
Removed the need for recorded meetings.